app/mailers/school_mailer.rb

```diff
@@ -446,7 +446,7 @@ def fip_provider_has_withdrawn_a_participant(withdrawn_participant:, induction_c
         name: induction_coordinator.user.full_name,
         withdrawn_participant_name: withdrawn_participant.user.full_name,
         school: withdrawn_participant.school.name,
-        delivery_partner: partnership.delivery_partner.name,
+        delivery_partner: partnership&.delivery_partner&.name || "No delivery partner",
```

Comment:
`Partnership` already has:
`delegate :name, to: :lead_provider, allow_nil: true, prefix: true`

We might add similar delegation method for deliver_partner so this line could end up being:
`delivery_partner: partnership.delivery_partner_name || "No delivery partner",`